index.css

```diff
@@ -504,3 +567,370 @@ textarea {
 .formato {
   height: 400px;
 }
+
+@media (min-width: 768px) {
+  nav {
+    display: none;
+  }
+
+  .active {
+    color: #fff;
+    width: 92px;
+    height: 20px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 20px;
+    text-decoration: none;
+    margin: 28px 0 0 120px;
+  }
+
+  .topnav {
+    display: flex;
+    justify-content: space-between;
+    color: #fff;
+    background: #3e3c3c url(Desktop/mail.png) 91% 62% no-repeat;
+    height: 92px;
+  }
+
+  .topnav #myLinks {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    color: #f2f2f2;
+    padding: 14px 50px 0 0;
+    margin-right: 110px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .roger {
+    height: 870px;
+    display: flex;
+    padding: 0 0 0 37px;
+    background: #1c1a19 url(Desktop/muñeco2.png) 100% 125% no-repeat;
+  }
+
+  .pad {
+    width: 566px;
+    height: 160px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-size: 50px;
+    font-weight: 400;
+    line-height: 82px;
+    margin-top: 130px;
+    margin-left: 80px;
+    color: #f55800;
+  }
+
+  .supporttest {
+    color: #fff;
+    width: 566px;
+    height: 24px;
+    margin: 20px 0 0 90px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .developer {
+    margin: 30px 0 0 0;
+  }
+
+  .theme1 {
+    color: #f4f5f7;
+    width: 440px;
+    height: 123px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .palet {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+  }
+
+  .palet a {
+    position: relative;
+    bottom: 270px;
+  }
+
+  .palet li {
+    height: 45px;
+  }
+
+  .myresent {
+    display: flex;
+    margin-left: 120px;
+    background-image: url(../Desktop/line3.png);
+    background-repeat: no-repeat;
+    background-position: 80% 75%;
+    background-size: 63%;
+  }
+
+  footer h2 {
+    width: 450px;
+    height: 256px;
+    font-style: normal;
+    font-family: "Inter", sans-serif;
+    text-align: start;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    color: #172b4d;
+    margin: 0  0 0 150px;
+  }
+
+  .myresent h2 {
+    height: 52px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 110px;
+    background-image: none;
```

Comment:
- [ ] As the last reviewer mentioned Please fix the margin between the project image and the text elements because this margin changes with different view widths in your design.

![image](https://user-images.githubusercontent.com/68796232/160047824-bccf11cc-0527-493e-b9c5-3695f9110e05.png)

![image](https://user-images.githubusercontent.com/68796232/160047756-0ba71686-a137-40fe-8d21-fb2cfaa5a1b9.png)

Reply:
think should be ok now
dimension 1440
![fit1](https://user-images.githubusercontent.com/74432345/160193839-fd67b985-a5fd-4a97-93c6-d652efb18e80.JPG)
romdom
![fit2](https://user-images.githubusercontent.com/74432345/160193922-0ccad1e0-9eaa-497f-87f1-8734dffa3cf6.JPG)

Reply:
thank you for your CR